### PR TITLE
Docs readability

### PR DESCRIPTION
**Pull request by fbs**
These very long one liners can be a bit difficult to read. Breaking them
up improves readability while still allowing copy/paste.

### CODE REVIEW

**Comment by brendangregg** [comment]
There are several one-liners than need to be split, but this PR does too many. One of my goals was as much as possible to have the heading, contents, and next heading fit on one screen. When you have a wall of text with no headings on the screen (because the content section has become too long) people get lost. So I've kept the vertical range as small as possible. I also want people to be able to print out the reference guide and have it be the fewest pages possible, so that it is neither intimidating nor wasteful of the reader's time. Longer descriptions, examples, and discussions, can go elsewhere, including new files we put in /docs.

One-liners that are less than 105 characters (including the hash) don't need to be split, since they fit. Only longer than that, where you must use the horizontal scroll bar, should be split. I'm discounting narrower resolutions like phones since the reference guide is for when you are programming bpftrace, and you won't be doing that on your phone.

Eg:

```
# bpftrace -e 'tracepoint:syscalls:sys_enter_openat { printf("%s %s\n", comm, str(args->filename)); }'
```

does not need to be split, since in the ref guide it should all be visible (I know it doesn't fit here because github comments are narrow).

But this one:

```
# bpftrace --include linux/path.h --include linux/dcache.h -e 'kprobe:vfs_open { printf("open path: %s\n", str(((path *)arg0)->dentry->d_name.name)); }'
```

I'd split like this:

```
# bpftrace --include linux/path.h --include linux/dcache.h -e 'kprobe:vfs_open {
    printf("open path: %s\n", str(((path *)arg0)->dentry->d_name.name)); }'
```

That turns 1 line into 2. 4-char indent.

**Comment by fbs** [comment]
Makes sense. I've updated the PR @brendangregg 

Should we just reformat the whole reference guide to stick to 105 char wide lines? It won't make any difference to the html it generates but I think it improves readability on the CLI. It also makes it easier to spot new examples that go over the 105 char length.

File: docs/reference_guide.md [hunk 1]
```diff
@@ -204,13 +204,13 @@ iscsid is sleeping.
...
 
 ```
-     1	#!/usr/local/bin/bpftrace
-     1	#!/usr/bin/env bpftrace
-     2
-     3	tracepoint:syscalls:sys_enter_nanosleep
-     4	{
-     5	        printf("%s is sleeping.\n", comm);
-     6	}
+1 #!/usr/local/bin/bpftrace
```
Comment (danobi) [inline docs/reference_guide.md]: What's with the double 1's?

Comment (fbs) [reply docs/reference_guide.md]: I think it's to indicate that there are (at least) 2 shebang options and that you should only use one. But it is a bit confusing

**Comment by mmarchini** [comment]
> Should we just reformat the whole reference guide to stick to 105 char wide lines?

I'm in favor of this. There are also linters we could use to enforce it if we want:

- [remark-lint](https://github.com/remarkjs/remark-lint)
- [markdownlint](https://github.com/markdownlint/markdownlint)
- Other [markdownlint](https://github.com/DavidAnson/markdownlint)
- etc.

File: docs/reference_guide.md (hunk 1 above)
Comment (brendangregg) [reply docs/reference_guide.md]: Yeah, it's too confusing. I'd ditch the env version.

Comment (fbs) [reply docs/reference_guide.md]: Fixed

**Comment by fbs** [comment]
I've limited almost all text to 105 chars. There are a few exceptions, toc, table and some code snippets but I think those are better the way they're now.

**Comment by danobi** [comment]
Once we migrate to github actions, we can use this: https://github.com/marketplace/actions/markdownlint-cli

**Pull request merged**